docker/base/DockerfileC7

```diff
@@ -0,0 +1,85 @@
+FROM centos:7
```

Comment:
Since we will keep adding more linux distros, I think it would be a bit better to differentiate the bistro using different subdirectory names and keep the same `Dockerfile` as the file name.

Perhaps `docker/base/centos7` and `docker/base/centos8` for now?

@SteVwonder ?

Reply:
I agree.  It would also be good to build both sets of base images as part of the CI.

Reply:
comments are inline in the last commit